missed an IE mention / centering using margin (#34174)
mdn/content: files/en-us/web/css/margin/index.md

### Horizontal centering

You can horizontally center an element within its parent by setting `margin: 0 auto;`.

> More commonly, though, people horizontally center element by setting {{cssxref("display")}}`: flex;` {{cssxref("justify-content")}}`: center;` on a container, which [centers its flex item](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container) children.

Reviewer: - How about the phrasing below (we can drop "to center an element horizontally" to avoid it repetition from the previous para)
- The quotes after the cssxref are adding an extra space before the ":". We can either remove the cssxref or link the entire property value pair.
```suggestion
A more common method to center an element horizontally is by setting `display: flex;` and `justify-content: center;` on a container, which [centers its flex item](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container) children.
```

> More commonly, though, people horizontally center element by setting {{cssxref("display")}}`: flex;` {{cssxref("justify-content")}}`: center;` on a container, which [centers its flex item](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container) children.

Author: ```suggestion
A more common method to center an element horizontally is by setting `display: flex;` and [`justify-content: center;`](/en-US/docs/Web/CSS/justify-content) on a container, which [centers its flex item](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container) children.
```
